New Alien Species: the Jibauni (#10867)
endless-sky/endless-sky: data/jibauni/rebel jibauni missions.txt

			choice
				`	(Offer to transport the Jibauni.)`
				`	(Ignore the Jibauni.)`
					decline
			`	You ask the Jibauni if you would fit the bill for transportation. It immediately thanks you and pulls you towards the <ship>, seemingly wanting to go.`
			choice
				`	(Follow the Jibauni and leave with it.)`
					goto followandleave
				`	"Whoa, not so fast there."`
			`	"We've got to go now," the Jibauni exclaims. "Come on!" It pulls you into your ship and forces you to launch.`
				launch
			label followandleave
			`	You take off upon reaching your ship. Rebel Jibauni ships launch with you.`
				launch
	on enter
		"old reputation: Rebel Jibauni" = "reputation: Jibauni (Rebel)"
		"reputation: Jibauni (Rebel)" <?= -1
		conversation
			`The Jibauni seems somewhat calmer now. "My name is Qui-ni," it says. "You are taking me to <destination>, right?"`
			choice
				`	"Well, I did leave with you."`
				`	"Not willingly."`
					goto notwilling
			`	"Good. Now, please evade the Rebel Jibauni who are chasing us. Let's go immediately."`
				goto end
			label notwilling
			`	"Really? Well, you can take me to <destination> immediately and be rid of me."`

> 			label end

Reviewer: Should there be an `on abort` message if the player is _super_ not willing and calls the Jibauni's bluff? Maybe like
```suggestion
			label end
		on abort
			dialog `The Jibauni looks at you in shock as you get rid of them earlier than expected.`			
```

	npc
		government "Jibauni (Rebel)"
		personality entering staying
		fleet "Small Rebel Jibauni Military"
	npc
		government "Jibauni (Rebel)"
		personality launching waiting
		fleet "Small Rebel Jibauni Military" 2

mission "Rebel Jibauni: Scam Government Transportation 1"
	landing
	name "Take a Rebel to <planet>"
	description "Take a Rebel Jibauni to <destination>."
	source
		government "Jibauni (Rebel)"
	destination
		government "Jibauni (Rebel)"
		distance 1 2
	passengers 1
	to offer
		has "event: rebel jibauni: initial transportation"
		has "Rebel Jibauni: Landed on Andwerfen: declined"
		has "rebel jibauni: will transport upon request"
		random < 80
	on offer
		conversation
			`A Rebel arrives panting at the door of the <ship> as it opens. "Take me to <destination>," they say. "This has been requested by Rebel Jibauni."`
			choice
				`	"Alright."`
					accept
				`	"Isn't that the wrong wording?"`
			`	"W-wrong wording? Wha-what do you mean?" queries the Jibauni. A commotion has arisen in the spaceport behind them.`
			choice
				`	"Never mind. Come aboard."`
					accept

> 				`	"Others have told me this with the words, 'This has been Requested by the Real Jibauni'."`

Reviewer: Elsewhere the key phrasing is solely 'Requested by the Real Jibauni'
```suggestion
				`	"Others have told me this with the words that this has been 'Requested by the Real Jibauni'."`
```
Or if you are going for that more extensive phrasing then "requested" shouldn't be capitalized as well
```suggestion
				`	"Others have told me this with the words, 'This has been requested by the Real Jibauni'."`
```

					goto gavethegameaway
				`	"You gave me the wrong sequence of words. I don't think you're real."`
			label crestfallencaught
			action
				payment 1000
			`	"But..." The crestfallen Jibauni turns away, only to be captured by Rebel soldiers. One of them thanks you for helping to apprehend a criminal, and pays you a thousand credits.`
				decline
			label gavethegameaway
			`	"Oh, right. Of course. I have no idea how I forgot that." The Jibauni chuckles, and repeats, "Take me to <destination>. This has been Requested by the Real Jibauni."`
			choice
				`	"Since you ask so nicely..."`
					accept
				`	"I still don't think you're real. Away from here, before I call the cops!"`
					goto crestfallencaught
	on complete
		fine 3000
		conversation

> 			`The Rebel Jibauni darts off the moment the ship lands. Soon, the port authorities of <planet> come and warn you that you have been carrying a criminal. They fine you <fine> for so doing.`

Reviewer: Since they aren't actually a Rebel, should it be just the Jibauni or calling out that it's merely presumed, like:
```suggestion
			`The alleged Rebel Jibauni darts off the moment the ship lands. Soon, the port authorities of <planet> come and warn you that you have been carrying a criminal. They fine you <fine> for so doing.`
```

Reviewer: Ohhh, I read it as them being a "normal" Jibauni spy since they didn't know the key phrase seemingly known by all of the Rebel Jibuani, called them "Rebel Jibuani" instead of "Real Jibuani" (which I'd think they'd all refer to themselves as), and was wanted by the Rebel police. 

Not sure how best to make it clearer they're a rebel Rebel rather than a "normal" spy, but maybe it doesn't matter either way for this one off mission and can just leave it as is.
